### PR TITLE
Lore friendly Quarg Rep

### PR DESCRIPTION
**Pull request by Ajm219**
During the Wanderers campaign, the Quarg mention their dislike for the Pug, and the Pug express similar feelings. When talking to the Coalition/Heliarchs, they mention their dislike for the Quarg, and you can assume that the feeling is mutual. With this in mind, I changed the Quarg/Pug and Quarg/Coalition/Heliarch so that killing Pug, gives a small rep increase to Quarg, and visa versa, and killing Quarg increases Coalition/Heliarch rep and visa versa.

### CODE REVIEW

File: data/governments.txt [hunk 1]
```diff
@@ -352,3 +352,639 @@ government "Wanderer"
...
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+government "Alpha"
```
Comment (tehhowch) [inline data/governments.txt]: ? You said this PR was about Quarg and lored enemies... why is there an entire redefinition of all governments?

Comment (Ajm219) [inline data/governments.txt]: The only reason its like that I pasted my goverments over the master, the only changes  are the ones listed above. Unless im wrong.

**Comment by Hadron1776** [comment]
@Ajm219 maybe pasting OVER `governments.txt` would help - github only counts the changed lines

File: data/governments.txt (hunk 1 above)
Comment (Hadron1776) [inline data/governments.txt]: Try using Unix style line endings. Codeblocks helps with that.

**Pull request closed without merging**